index.bs

```diff
@@ -1468,19 +1512,186 @@ To <dfn>validate a suffix</dfn> |suffix|, run the following steps:
 
 </div>
 
+### Starting Directory ### {#api-filepickeroptions-starting-directory}
+
+<div class="note domintro">
+The {{FilePickerOptions/id}} and {{FilePickerOptions/startIn}} fields can be specified to suggest
+the directory which the file picker opens to.
```

Comment:
Sentence structure here is awkward in English.

How about: "the directory in which the file picker opens."